erpnext_com/www/docs/user/manual/en/customize-erpnext/custom-scripts/filter-options-in-select-field.md

```diff
@@ -0,0 +1,24 @@
+<!-- add-breadcrumbs -->
+# Filter Options in Select Field
+
+Lets say you have a field named state of type [Select](/docs/user/manual/en/customize-erpnext/articles/field-types) and another field named city of type [Select](/docs/user/manual/en/customize-erpnext/articles/field-types). State has two values Karnataka and Maharashtra and City has four values, Bangalore, Mysore, Mumbai and Pune. If you would like to filter options in city field based on the values chosen in state field, you can write custom script as shown below.
```

Comment:
What are you trying to say by 'state of type Select'?

Reply:
its a field with name state and its of type select(dropdown)...

Reply:
Updated the PR...